sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsReflector.java

```diff
@@ -36,7 +36,8 @@ private PipelineOptionsReflector() {}
    *
    * @see PipelineOptionsReflector#getOptionSpecs(Iterable)
    */
-  static Set<PipelineOptionSpec> getOptionSpecs(Class<? extends PipelineOptions> optionsInterface) {
```

Comment:
Is this change needed for supporting mountTempDir?

Reply:
Yes. Otherwise mountTempDir option will not be exposed to Python SDK since it's marked as hidden.